zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -329,24 +332,49 @@ where
 /// If the data in an address is invalid, this function can:
 /// - modify the address data, or
 /// - delete the address.
-//
-// TODO: re-enable this lint when last_seen_limit is used
-#[allow(unused_variables)]
+///
+/// Currently, this method will offset the reported `last_seen` time to prevent clock skews
+/// from causing the peers to be placed too far back or in the front of the reconnection queue
+/// incorrectly.
```

Comment:
1. We try to format security conditions with a header
2. We only adjust times that are in the future
3. Focus on the security action and the security impact

```suggestion
/// # Security
///
/// Adjusts untrusted last seen times so they are not in the future. This stops
/// malicious peers keeping all their addresses at the front of the connection
/// queue. Honest peers with future clock skew also get adjusted.
```